diesel_tests/tests/types_roundtrip.rs

```diff
@@ -186,6 +186,12 @@ mod pg_types {
         (u8, u8, u8, u8, u8, u8),
         mk_macaddr
     );
+    test_round_trip!(
+        macaddr8_roundtrips,
+        MacAddr8,
+        (u8, u8, u8, u8, u8, u8, u8, u8),
+        mk_macaddr
```

Comment:
This `mk_macaddr` is a function below in this file. We need to have a `mk_macaddr8` variant there as well.